actors/miner/src/lib.rs

```diff
@@ -4025,7 +4025,7 @@ where
         ));
     }
 
-    Ok(resolved)
+    Ok(Address::new_id(resolved))
```

Comment:
Probably just return the ActorID here, unless that causes lots of churn at caller.s